libs/database/tables/load_functions.c

```diff
@@ -0,0 +1,99 @@
+/*
+** EPITECH PROJECT, 2022
+** Teams
+** File description:
+** read_functions
+*/
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+
+char *get_file_content(const char *filepath)
+{
+    char *buffer = 0;
+    long length;
+    FILE *f = fopen (filepath, "rb");
+
+    if (f) {
+        fseek (f, 0, SEEK_END);
+        length = ftell (f);
+        fseek (f, 0, SEEK_SET);
+        buffer = calloc(sizeof(char), length + 1);
+        if (buffer) {
+            fread (buffer, 1, length, f);
+        }
+        fclose (f);
+    }
+    return (buffer);
+}
+
+char *get_arg(char *entity, size_t *cur)
```

Comment:
cur ???

Reply:
cursor (that's for the Epitech C Coding Style)